CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/common?activeTab=versions
 
+## v0.31.2
+
+04-02-2019 14:46 PDT
+
+## Bug Fixes
+
+- fix: @types/node@11.13.0 breaks paginate/promisifyAll ([#397](https://github.com/googleapis/nodejs-common/pull/397))
+- fix(ts): do not ship @types/duplexify ([#393](https://github.com/googleapis/nodejs-common/pull/393))
+- fix(deps): bump the min version required for all deps
+
+### Implementation Changes
+
+- refactor: remove configurable request ([#394](https://github.com/googleapis/nodejs-common/pull/394))
```

Comment:
This seems like a breaking change? @JustinBeckwith yes/no/maybeso?

Reply:
the original pull request indicates so:

```
BREAKING CHANGE: This PR removes the ability to configure a custom implementation of the Request module. This was necessary when we were migrating from request to teeny-request, but that migration is now complete. All interfaces at accepted a custom implementation of request will no longer accept one. teeny-request is now just included in the box.
```

Reply:
(related, oh my goodness, can't wait to automate this process).

Reply:
Oh man, I should have read the extended message